packages/code-du-travail-ui/__tests__/Alert.spec.tsx

```diff
@@ -0,0 +1,54 @@
+//
+
+import React from "react";
+import { create } from "react-test-renderer";
+import { Alert } from "src/components/Alert";
+
+test("<Alert>Alert !</Alert> should render an Alert component", () => {
+  const component = create(<Alert>Alert !</Alert>);
+  expect(component.toJSON()).toMatchSnapshot();
+});
+
+test("<Alert primary>Primary</Alert> should have 'alert__primary' class", () => {
```

Comment:
test label is misleading since we make no assertion on className but on snapshot content, 
if classnames change, snapshot can be updated automatically, thus relying on className force us to update all test name.

Reply:
?


> test label is misleading since we make no assertion on className but on snapshot content,

The assertion is made on everything inside a snapshot no ?

> if classnames change, snapshot can be updated automatically, thus relying on className force us to update all test name.

It's not because you can update all snapshot in one go that you should.
Snapshot are test content too. they can/should be edited by hand when necessary 

---

How would you test this ?

Reply:
Sorry my comment was misleading
I think we should use  
```
 test("<Alert primary>Primary</Alert> should render, () => { 
   expect(component.toJSON()).toMatchSnapshot();
})
```
rather than use implementation detail (eg class name we use for alert